client/sections.js

```diff
@@ -22,21 +22,20 @@ const extensionSections = extensions.map( extension => {
 	}
 } );
 
-if ( config.isEnabled( 'devdocs' ) ) {
-	sections.push( {
-		name: 'devdocs',
-		paths: [ '/devdocs' ],
-		module: 'devdocs',
-		secondary: true,
-		enableLoggedOut: true
-	} );
+sections.push( {
```

Comment:
@mtias yes, the bundle will be generated for any envs build. Or more correctly, this should ideally be bringing us to a single build